packages/xo-server/src/api/backup-ng.mjs

```diff
@@ -380,3 +380,41 @@ fetchFiles.params = {
     type: 'string',
   },
 }
+
+export function mountPartition({ remote, disk, partition }) {
```

Comment:
I think we should add a timeout or autoDismount timer ( like 24h) to ensure we don't lock backup in place